pkg/sysregistries/system_registries.go

```diff
@@ -39,9 +39,9 @@ func normalizeRegistries(regs *registries) {
 	}
 }
 
-// Reads the global registry file from the filesystem. Returns
+// ReadRegistryConf reads the global registry file from the filesystem. Returns
 // a byte array
-func readRegistryConf(sys *types.SystemContext) ([]byte, error) {
+func ReadRegistryConf(sys *types.SystemContext) ([]byte, error) {
```

Comment:
Why make this public?

Reply:
That way buildah or podman could give the location of the registries.conf message.

podman pull foobar
foobar image not found in any registry,  you can add registries to /etc/containers/registries.conf

Reply:
If you mean the path, that’s available through the newly added `RegistriesConfPath` below. This returns the _contents_ of the file, without revealing the path at all.